src/components/auth/AuthHeader.tsx

```diff
@@ -0,0 +1,43 @@
+import Link from "next/link";
+import {useAppContext} from "@/context/AppContext";
+import {UserButton} from "@clerk/nextjs";
+import Spinner from "@/app/loading";
+import React from "react";
+
+const AuthHeader = () => {
+    const {isLoggedIn, isLoading} = useAppContext();
+
+    if (isLoading) {
+        return <Spinner/>
+    }
+
+    return (
+        <>
+            {!isLoggedIn ? (
+                <div className="flex items-center gap-3 sm:gap-3">
+                    <Link href="/login">
+
+                        <button
+                            className="inline-flex items-center justify-center gap-1.5 rounded-lg border border-gray-200 px-5 py-3 text-gray-500 transition hover:bg-gray-50 hover:text-gray-700 focus:outline-none focus:ring"
+                            type="button"
+                        >
+                            <span className="text-sm font-medium">Login </span>
+                        </button>
```

Comment:
Would it be better to make use of existing Button component so that we have consistent design?